src/components/member/tabs/quote-tab/quote-sub-section.tsx

```diff
@@ -20,16 +20,11 @@ export const QuotesSubSection: React.FC<{
 }> = ({ memberId, contractType, quotes }) => {
   const [isWip, setIsWip] = React.useState(false)
   const activeQuotes = quotes.filter((quote) => !isSignedOrExpired(quote))
-  const signedQuotes = quotes.filter(isSigned)
-  const hasActiveQuotes = activeQuotes.length > 0
-  const hasSignedQuotes = signedQuotes.length > 0
 
   return (
     <Wrapper>
-      {!hasActiveQuotes && !hasSignedQuotes && (
-        <Button onClick={() => setIsWip(!isWip)}>Create</Button>
-      )}
-      {!hasActiveQuotes && !hasSignedQuotes && isWip && (
+      {<Button onClick={() => setIsWip(!isWip)}>Create</Button>}
```

Comment:
We really don't want IEX to create quotes if the contract is active, can we at least check that it is terminated before allowing this button?
```suggestion
      {(!hasSignedQuotes || contract.terminationDate) && <Button onClick={() => setIsWip(!isWip)}>Create</Button>}
```